nixos/modules/config/console.nix

```
@@ -46,7 +46,7 @@ in
 
     font = mkOption {
       type = with types; either str path;
-      default = "Lat2-Terminus16";
+      default = "";
```

What is the default font if linux is older than 5.3 and we have set this to an empty string?

---

This would restore the original behavior but I don't know why `Lat2-Terminus16` was chosen in the first place?
```suggestion
      default = lib.optionalString (versionOlder config.boot.kernelPackages.kernel.version "5.3") "Lat2-Terminus16";
      defaultText = "";
```

---

I just read your comments. That cleared things up.

---

Maybe the doc string should be updated to clarify this.

---

Yeah that would be helpful.

---

Reading the manual for setfont, I found this:
>The default font is a file default (or default8xN if the -N option was given for some number N) perhaps with suitable extension (like .psf).

And indeed, a default8x16 font does exist. It might be that this font is identical to the kernel's own default. In this case, it might be better (or even necessary) to handle a null value here, and make that the default. This SE question seems to agree: https://unix.stackexchange.com/questions/710337.
My bad for submitting this without actually testing everything. :stuck_out_tongue: